domain/consensus/model/interface_processes_transactionvalidator.go

```diff
@@ -8,6 +8,8 @@ import (
 // it's possible to determine whether a transaction is valid
 type TransactionValidator interface {
 	ValidateTransactionInIsolation(transaction *externalapi.DomainTransaction) error
+	ValidateTransactionWithContextAndWithoutUTXOContext(stagingArea *StagingArea, tx *externalapi.DomainTransaction,
```

Comment:
Consider renaming this to ValidateTransactionInContextIgnoringUTXO, so that we won't have a new concept called "utxo context"